projectq/backends/_ionq/_ionq_http_client.py

```diff
@@ -241,18 +248,17 @@ def _handle_sigint_during_get_result(*_):  # pragma: no cover
         raise RequestTimeoutError("Timeout. The ID of your submitted job is {}.".format(execution_id))
 
 
-def show_devices(verbose=False):
-    """Show the currently available device list for the IonQ provider.
+    def show_devices(self, verbose=False):
```

Comment:
The `verbose` argument is never used in this method. Is it really required or can we remove it since this is now using the attribute of the `IonQ`class?

Reply:
Ah, great point! We should be able to rely on the class attribute entirely now.